hyperspace/core/src/command.rs

```diff
@@ -48,6 +48,11 @@ pub enum Subcommand {
 	CreateConnection(Cmd),
 	#[clap(name = "create-channel", about = "Creates a channel on the specified port")]
 	CreateChannel(Cmd),
+	#[clap(
+		name = "sync-clients",
+		about = "Tries to sync light clients on both chains to the latest"
+	)]
+	SyncClients(Cmd),
```

Comment:
Why can't we just use `relay`?

Reply:
The relay command listens for new updates, it doesn't necessarily fetch old updates.

Reply:
But isn't this PR about making `relay` fetch old updates?

Reply:
I agree. Usage-wise, it's necessary to merge both functionalities.

Reply:
Are you going to merge the functionality @Wizdave97 ?

Reply:
Yeah I can merge both

Reply:
@blasrodri @Wizdave97 I mean, we don't need `SyncClients` if `Relay` will be submitting headers from the point it stopped. Can we just call `sync_clients` before the `relay` function if the second won't send blocks from the past?

Reply:
Yeah that's the approach I'm taking

Reply:
Let's remove the command then and just use the function inside `Relay` command